src/nth.js

```diff
@@ -8,14 +8,16 @@
  */
 
 /**
- * Returns `nth` (starts from `0` of course) item of an `iterable`.
+ * Returns `nth` (starts from `0` of course) item of the given `iterable`.
+ * Consumes all items of the generator.
+ * Consumes all items up to the index of the iterator, including the returned item.
```

Comment:
These tests are showing the difference: https://github.com/ckeditor/ckeditor5-utils/pull/138/files#diff-7b7a935b05c5a95c53b4e26cadf83040R30